server/server.js

```diff
@@ -2,11 +2,13 @@ require('dotenv').config()
 const express = require('express');
 const google = require('./routes/google');
 const dropbox = require('./routes/dropbox');
+const indexRouter = require('./routes/index');
 
 const app = express();
 const port = process.env.PORT || 3001;
 
 app.use('/google', google);
 app.use('/dropbox', dropbox);
+app.use('/', indexRouter);
```

Comment:
Maybe we nest this under `/api` and call it `apiRouter`? That way, we won't have to be careful about order here, or prefix all the API routes.

Reply:
That makes sense. Will update.